tests/common/plugins/sanity_check/checks.py

```diff
@@ -323,9 +323,63 @@ def get_arp_pkt_info(dut):
     return intf_mac, mgmt_ipv4
 
 
+def mux_sim_check_downstream(active_tor, standby_tor, ptfadapter,
```

Comment:
To keep it consistent with upstream function and callers, the args `active_tor, standby_tor` should be rename to `upper_tor_host, lower_tor_host`.

Reply:
Do we have testcase where active and standby can be switched? If the argument is to specify role, then I say keep it. If it is to specify name/location, then change accordingly.

Reply:
For upstream, active and standby roles do not matter since upstream traffic is broadcast. But active/standby does matter for downstream, which is why I changed the argument names, since this function is not aware of upper/lower relationship, it only cares about active/standby.